includes/class-wc-data-store.php

```diff
@@ -28,8 +28,10 @@ class WC_Data_Store {
 	 * Ran through `woocommerce_data_stores`.
 	 */
 	private $stores = array(
-		'coupon'        => 'WC_Coupon_Data_Store_CPT',
-		'payment-token' => 'WC_Payment_Token_Data_Store_Table',
+		'customer'         => 'WC_Customer_Data_Store',
```

Comment:
Should we suffix this one with `_Table` or maybe something to show it using WP's user system, or something else to signify it's storage?